src/rime_api.cc

```diff
@@ -167,6 +167,11 @@ RIME_API Bool RimeSyncUserData() {
   return Bool(deployer.StartMaintenance());
 }
 
+RIME_API Bool RimeSyncUserDict() {
+  Deployer& deployer(Service::instance().deployer());
+  return Bool(deployer.ScheduleTask("user_dict_sync"));
```

Comment:
Is this safe to do while user input session accesses user dictionary simutaniously?